qemu/tests/cpu_model_negative.py

```diff
@@ -0,0 +1,52 @@
+import re
+import logging
+
+from avocado.utils import process
+
+from virttest import cpu
+from virttest import error_context
+from virttest import utils_qemu
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Runs CPU negative test:
+
+    1. Boot guest with improper cpu configuration
+    2. Verify guest boot fail
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+
+    if ('enforce' in params['shortname'] and
+            '+avx512_bf16' in params['cpu_model_flags']):
+        if 'avx512_bf16' in cpu.get_host_cpu_models():
+            test.cancel('This case only test on the host without the flag'
+                        ' avx512_bf16.')
+
+    qemu_bin = utils_misc.get_qemu_binary(params)
+    if 'OUT_OF_RANGE' in params['wrong_cmd']:
+        machine_type = params['machine_type'].split(':')[-1]
+        m_types = utils_qemu.get_machines_info(qemu_bin)[machine_type]
+        m_type = re.search(r'\(alias of (\S+)\)', m_types)[1]
+        max_value = utils_qemu.get_maxcpus_hard_limit(qemu_bin, m_type)
+        smp = str(max_value + 1)
+        params['wrong_cmd'] = params['wrong_cmd'].replace(
+            'MACHINE_TYPE', machine_type).replace('OUT_OF_RANGE', smp)
+        msg = params['warning_msg'].replace('SMP_VALUE', smp).replace(
+            'MAX_VALUE', str(max_value)).replace('MACHINE_TYPE', m_type)
+        params['warning_msg'] = msg
+    warning_msg = params['warning_msg']
+    wrong_cmd = '%s %s' % (qemu_bin, params['wrong_cmd'])
+    status, output = process.getstatusoutput(wrong_cmd)
```

Comment:
Can we add one log to print what's the `output` in case we get one warning which is not expected?